### ISSUE TITLE
[Bug]: Getting error while trying to use plugin cypress-plugin-visual-regression-diff (please, details in description)

### ISSUE THREAD

**Issue by sakilandeswari** [FRSOURCE/cypress-plugin-visual-regression-diff#308]
### Describe the Bug

while running cypress component with plugin cypress-plugin-visual-regression-diff, getting below error

command executed: npx nx run-many -t component-test -p bindings-ng-pos  --parallel=false --skip-nx-cache --watch

Error
taskcp-visual-regression-diff-getScreenshotPathInfo, Object{4}, {log: false}
AFTER ALL (FAILED)
1
taskcp-visual-regression-diff-cleanupImages, {log: false}
CypressError
cy.task('cp-visual-regression-diff-cleanupImages') failed with the following error:

The 'task' event has not been registered in the setupNodeEvents method. You must register it before using cy.task()

Fix this in your setupNodeEvents method here:

### Steps to reproduce

Followed steps mentioned in the below page
https://github.com/FRSOURCE/cypress-plugin-visual-regression-diff

### Expected behavior

Visual comparison should work fine without any issue 

no reference in document to resolve this issue

### Package version

3.3.10

### Device

Chrome Browser - Mac

### OS and version

Mac OS Sonama 14.1.2 (23B92)

### Browser and version

Chrome 120.0.6099.109

### Node version

20.8.0

### I can work on it by myself

- [ ] Yes
- [X] No

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by dsuchan** [comment]
I don't know if this could be related, but i get the same error message in my cypress app, where i don't initialize the node event. I'm working in a big repo, where we have multiple e2e apps which some of them uses visual regression testing and some of them don't. The apps that doesn't need to use this feature doesn't have it initialized in `setupNodeEvents`. When the test finishes, it fails inside afterAll hook on exactly the same message. Is there any settings i need to use to skip this task if i don't want to use that?

**Comment by FRSgit** [comment]
Hello everyone!

Sorry for long delay in the response.
Please help me understand the issue. From what I know about the plugin inner workings. When you do the second step from the [installation guide](https://github.com/FRSOURCE/cypress-plugin-visual-regression-diff?tab=readme-ov-file#installation), this plugin sets up a global [after hook](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests#Hooks) in which it tries to automatically cleanup unused image files.
But, for this feature to work a cleanup task needs to be registered and the task registration happens only when you use `initPlugin` function within `setupNodeEvents` in your cypress.config file.

So, to sum up - if you do only first two steps of installation guide, you'll receive this error. And I guess that shouldn't be considered a bug?
But I might be wrong, so please give me more info about your scenarios for me to understand where the plugin/documentation fails.

**Comment by JakeMetz8** [comment]
I receive the same error message and have followed the install instructions in the readme. Cypress then seems to enter an infinite loop as described in [issue #323](https://github.com/FRSOURCE/cypress-plugin-visual-regression-diff/issues/323). 

I have not set the env `pluginVisualRegressionCleanupUnusedImages` either.

![Image](https://github.com/user-attachments/assets/132d2a5d-6af0-49ba-9682-6a1276dc554c)

**Comment by FRSgit** [comment]
I didn't get an answer from the original author for 2 years now, so closing this one.
